packages/main/cypress/specs/Toolbar.cy.tsx

```diff
@@ -321,6 +321,96 @@ describe("Toolbar general interaction", () => {
 			.eq(3)
 			.should("be.focused");
 	});
+
+	it("Should render ui5-button by toolbar template, when slotting ui5-toolbar-button elements", () => {
+		cy.mount(
+			<Toolbar>
+				<ToolbarButton 
+					icon="decline" 
+					stableDomRef="tb-button-decline"
+					overflowPriority="NeverOverflow" 
```

Comment:
We should add one more assertion related to NeverOverflow priority and button not disappearing in smaller screen size.